### ISSUE TITLE
(Some) old style avatars are not visibile inside the AAQ forums

### ISSUE THREAD

**Issue by emilghittasv** [mozilla/sumo#2783]
**Describe the bug**
The avatar of the OP user in https://support.mozilla.org/en-US/questions/1563564 is not visible falling back to the alt text

**Comment by denyshon** [comment]
This issue is caused by our [JS-level avatar fallback](https://github.com/mozilla/kitsune/blob/main/kitsune/sumo/static/sumo/js/profile-avatars.js) being limited to `img.avatar` (images with the "avatar" class applied). On question details pages (as well as some others), avatar images do not have the "avatar" class and are instead located inside other elements with the "avatar" class.

In this particular case, the OP _has_ an avatar link (https://firefoxusercontent.com/b54ab6047b54c1ad6140b7a25ed8c23c), but it is broken. This leaves the JS-level fallback as the only fallback option, but this fallback doesn't happen on the question details page due to our selector.

**[Testing]** Steps to reproduce should be the following:
1. Create a user whose profile has a non-image link in `fxa_avatar` (but the value must be non-empty for the [Python-level fallback](https://github.com/mozilla/kitsune/blob/9ed099fc9cec9ef7a908a0d21d20fd19194d9995/kitsune/users/templatetags/jinja_helpers.py#L30-L37) to pass).
2. Go to their profile page (view profile as another user) and make sure the default fallback avatar is displayed (current state).
3. Post a question as that user and go to the question page (as any user, probably?). Verify that the default fallback avatar is not displayed without the patch, and is displayed with the patch applied.
4. (Optional) Go through other pages where avatars are displayed (after creating a forum post, after getting a KB revision approved, after assigning a badge, etc.) and verify that with the patch applied, the default fallback avatar is always displayed. I believe that extending the selector to child images of elements with the "avatar" class should cover all the cases, but I haven't explicitly checked that.

**Comment by emilghittasv** [comment]
@denyshon after doing some exploratory testing I have noticed that this issue is partially fixed and filled https://github.com/mozilla/sumo/issues/2795 for the remaining/identified areas.

I can confirm that this issue is no longer reproducible inside the AAQ forums.

I am going to move this particular ticket inside the `release` column.
